zilliqa/src/message.rs

```diff
@@ -87,7 +87,7 @@ impl Vote {
     ) -> Self {
         let mut bytes = Vec::new();
         bytes.extend_from_slice(block_hash.as_bytes());
-        bytes.extend_from_slice(&view.to_be_bytes());
+        //bytes.extend_from_slice(&view.to_be_bytes());
```

Comment:
Why did we remove the view from vote signatures?

Reply:
when you enable BLS everything needs to be signed on the same stuff essentially (which it isn't in main). This PR contains the full fix: https://github.com/Zilliqa/zq2/pull/607